Rakefile

```diff
@@ -1,138 +1,4 @@
-require "fileutils"
-require "rototiller"
-require "yard"
-require "rubocop/rake_task"
-require "flog_task"
-require "flay_task"
-require "roodi_task"
-require "rubycritic/rake_task"
+# FIXME: this might not work when specifying the rakefile from another path
```

Comment:
You could check to see if the directory exists, and if not recursively go up a directory and try again?

Reply:
actually.  the Gemfile breaks if you try to run rake in bundle exec from another dir.  
in here we could use `application.original_dir` to find the project root.  but not in the Gemfile.

i'm gonna remove the comment and we'll cross that bridge later